scripts/db-heal-test.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+###########################
+#--- Dynamic variables ---#
+###########################
+
+# Aida paths
+aida_path=''
+aida_db_path=''
+tmp_path=''
+
+# Block variables
+sync_block=20000
+kill_block=11000
+restore_block=10000
+final_block=21000
+
+##########################
+#--- Static variables ---#
+##########################
+
+carmen_root=$(cd ../go && pwd)
+
+
+################
+#--- Script ---#
+################
+
+COMMAND="./build/aida-vm-sdb substate --db-tmp "$tmp_path" --carmen-schema 5 --db-impl carmen --aida-db "$aida_db_path" --no-heartbeat-logging --track-progress --archive --archive-variant s5 0 "$sync_block""
```

Comment:
I was thinking about adding it, but first command (the interrupted one) actually does not make sense to add validation, does it?

Anyway, since it only slows it down by a forgettable time, we can just add it.

Reply:
Not sure, does `--validate` also enable `--validate-state-hash` ? The latter, I think, would be the strongest check that we got it right.

Reply:
`--validate` enables every validation